actionpack/test/dispatch/debug_exceptions_test.rb

```diff
@@ -178,6 +180,10 @@ def call(env)
     get "/parameter_missing", headers: { "action_dispatch.show_exceptions" => true }
     assert_response 400
     assert_match(/ActionController::ParameterMissing/, body)
+
+    get "/invalid_mimetype", headers: { "Accept" => "text/html,*", "action_dispatch.show_exceptions" => true }
```

Comment:
I don't think `*` is not an invalid mime type

Reply:
oops, * is but `*/*` is not and we got an exception for it

Reply:
that's because it was malformated as well 
<img width="268" alt="Screenshot 2019-03-25 at 22 13 54" src="https://user-images.githubusercontent.com/8122246/54954602-4d53f480-4f4b-11e9-88c0-6af711b2f8c3.png">

it was surrounded with spaces ` */* `

Reply:
To be fair I also thing `*` is valid, or at least IE10 sends it by default.

Reply:
hmm if IE10 sends it by default then we should probably support it, although the RFC doesn't say it's valid https://tools.ietf.org/html/rfc7231#page-38

I'll try to modify the regex

Reply:
What if the malformed accept header is being sent by a browser? We should not start to show error in that case. If we do that users of that browser can't access the app anymore.

Reply:
Ok, just confirmed is not a browser for the invalid accept header. Not sure about `*` though. I'm afraid it is IE10, but it could be a bot trying to pass by a regular browser.

Reply:
Thanks for checking (How did you check btw, so I know for next time).

> What if the malformed accept header is being sent by a browser?

Don't you think this argument can be applied to any issue related to the UA? If the browser sends malformated query paratemers or invalid bite sequence, rack would raise and block access to the app as well, no ?

Reply:
>(How did you check btw, so I know for next time).

All requests were to xml paths in very crazy pages. This looks like behavior of bots.


> If the browser sends malformated query paratemers or invalid bite sequence, rack would raise and block access to the app as well, no ?

Right, but this started because of a change on the browser, where now this started because of a change in the framework that is now rejecting a request that were valid for years.

Reply:
I had a look in our logs and DW, the Accept headers on IE 10 is correctly formatted and sent as '*/*'

Reply:
Just to confirm @rafaelfranca , do we want to modify the regex to include the `*` mime?

Reply:
If we know IE10 is not sending that, it is fine to exclude it.

Reply:
> If we know IE10 is not sending that, it is fine to exclude it.

For anyone running into the same situation, I believe we are getting this error from valid IE 10 machines.  Example UA below:

```Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.108 Safari/537.36```

I looked up the UA and it appears to be listed in https://developers.whatismybrowser.com/useragents/explore/software_name/chrome/ as well.

Maybe it would be good to allow `*`?